lib/collections/collections.js

```diff
@@ -425,7 +425,7 @@ var settings = new SimpleSchema ({
     radarrQUALITYPROFILEID: {
         type: Number,
         label: "Download quality profile for Radarr",
-        defaultValue: 1,
+        defaultValue: '1',
```

Comment:
Ok, the fix has been committed and pushed. https://github.com/lokenx/plexrequests-meteor/pull/480/commits/4f54ac3ce48d3cebb4e29e8ae2529e40e8d13987